gef-ui/server/controllers/application-details/index.js

```diff
@@ -1,57 +1,119 @@
 /* eslint-disable no-underscore-dangle */
 import _startCase from 'lodash/startCase';
-import * as api from '../../services/api';
-import { mapSummaryList, status } from '../../utils/helpers';
-import { exporterItems, coverItems, facilityItems } from '../../utils/display-items';
+import { mapSummaryList } from '../../utils/helpers';
+import {
+  exporterItems, coverItems, facilityItems,
+} from '../../utils/display-items';
 import { getUserAuthorisationLevelsToApplication } from '../../utils/user-authorisation-level';
-import { PROGRESS, FACILITY_TYPE, AUTHORISATION_LEVEL } from '../../../constants';
+import { FACILITY_TYPE, AUTHORISATION_LEVEL } from '../../../constants';
+
+import Application from '../../models/application';
+
+function buildHeader(app) {
+  if (!['Draft'].includes(app.status)) {
+    const main = {
+      ukefDealId: app.ukefDealId || '-',
+      submissionDate: app.submissionDate || '-',
+      companyName: app.exporter.details.companyName,
+      applicationShowSummary: ['SUBMITTED_TO_UKEF', 'CHANGES_REQUIRED', 'BANK_CHECK'].includes(app.status),
+      applicationStatus: app.status,
+      dateCreated: app.createdAt,
+      timezone: app.maker.timezone || 'UTC',
```

Comment:
We've historically been setting the timezone to 'Europe/London', is UTC a better way forward? Just thinking of potential confusion for future devs.

Reply:
Good spot - I'll update.